darwinia-network/darwinia-bridge-sol
# Lean BEEFY

- beefy e2e test script
- beefy types
- pangoro <=> bsctest bridge deploy script
- verify contract in bsctest
- change beefy threshold
- test beefy authority set change
- dapp.tools support evm_version and verify embed
- utils solidity unit test
- adjust dir
- code linter
- foundry support
- update docs
- lean BEEFY
- fix #139
